asllib/doc/Statements.tex

```diff
@@ -3105,27 +3188,27 @@ \subsubsection{ASTRule.SPrint}
 \subsection{Typing}
 
 \subsubsection{TypingRule.SPrint\label{sec:TypingRule.SPrint}}
-
 \paragraph{Prose}
 All of the following apply:
 \begin{itemize}
   \item $\vs$ denotes the print statement with arguments $\vargs$ and newline indicator $\vnewline$;
-  \item annotating each expression $\vargs_i$ in $\tenv$ yields $(\vt_i, \vargsp_i)$\ProseOrTypeError, for $i=1..k$;
-  \item checking that $\vt_i$ is a singular type yields $\True$\ProseOrTypeError, for each $i=1..k$;
+  \item annotating, \Proselistrange{$i$}{$\vargs$}, the expression $\vargs_i$ in $\tenv$ yields $(\vt_i, \vargsp_i, \vxs_i)$\ProseOrTypeError;
```

Comment:
why "annotating," rather than "annotating" (no comma)?

Reply:
this is to allow discussing the index and the range of indices.